src/utils/LinkedObject.hxx

```diff
@@ -36,15 +36,45 @@
 #ifndef _UTILS_LINKEDOBJECT_HXX_
 #define _UTILS_LINKEDOBJECT_HXX_
 
+#include <atomic>
+
 #include "utils/Atomic.hxx"
+#include "utils/Uninitialized.hxx"
+
+/// This object encapsulates an Atomic, which never gets destroyed. It is
+/// intended to be used only as static object, because we depend on zero
+/// initialization at construction time.
+class CreateOnlyAtomic
```

Comment:
Should this be a template class for the atomic type?

Reply:
actually this is an `utils/Atomic.hxx` (of which there is only one), not an std::atomic (which is a template).